master/docs/manual/installation/misc.rst

```diff
@@ -115,33 +115,55 @@ Upgrade pip, setuptools and wheel
    cd c:\Python39
    python.exe -m pip install --upgrade pip setuptools wheel
 
-Install, create and activate virtualenv
-```````````````````````````````````````
+Install, create and activate virtualenv [optional]
+``````````````````````````````````````````````````
+This step is optional and may depend on your needs.
```

Comment:
well, we highly recommend to use this and avoid messing up with your python install.
a virtualenv can easily be recreated, while a python install, it is much more annoying

Reply:
Hmm, the main installation page expresses it more as an option depending on comfort: https://github.com/buildbot/buildbot/blob/master/master/docs/manual/installation/installation.rst#installation-in-a-virtualenv

I thought this was being more consistent...